src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -66,6 +66,9 @@ public ObservableList<Person> getFilteredPersonList() {
     }
 
     @Override
+    public String getNextSession() {
+        return model.getNextSession();
+    }
     public ObservableList<TimeSlot> getTimeSlotList() {
```

Comment:
u deleted the @override anotation

Reply:
i could have sweared the merge conflict didnt show two override but mayb thats the issue like merge conflict thot its the same tag or smt, will fix!

Reply:
other than that ok?